.github/workflows/browserslist-db-update.yml

```diff
@@ -55,8 +59,8 @@ jobs:
 
       - name: Configure git
         run: |
-          git config --global user.email "actions@github.com"
-          git config --global user.name "GitHub Actions"
+          git config --global user.email "138160377+google-site-kit-bot@users.noreply.github.com"
+          git config --global user.name "google-site-kit-bot"
```

Comment:
Hmm... Reading the man page for Git (on my local Ubuntu 22.04), it suggests we won't need these any more, as the environment variables will override them anyway.

It doesn't hurt to keep them, but would be nice to tidy up if they are not required...

```
GIT_AUTHOR_NAME
    The human-readable name used in the author identity when creating commit or tag objects, or when writing reflogs. Overrides the user.name and author.name configuration settings.

GIT_AUTHOR_EMAIL
    The email address used in the author identity when creating commit or tag objects, or when writing reflogs. Overrides the user.email and author.email configuration settings.
```